go/pumiceDB/client/PumiceDBClient.go

```diff
@@ -93,132 +107,86 @@ func getPmdbReq(reqArgs *PmdbReqArgs) (unsafe.Pointer, int64) {
 	return reqPtr, reqLen
 }
 
-//Write KV from client.
-func (obj *PmdbClientObj) Write(reqArgs *PmdbReqArgs) (unsafe.Pointer, error) {
-
-	var rBytes bytes.Buffer
-	var err error
+func (r *PmdbReqArgs) Write() error {
+	return r.writeKV() // Perform the write
```

Comment:
The term `KV` should be removed from this file since the pmdb client is not dealing w/ 'keys' and 'values' directly.  `writeKV()` could be renamed to `writeInternal()`.

Reply:
changed the name.